tests/framework/v1alpha1/framework.go

```diff
@@ -51,15 +61,15 @@ var (
 
 // Framework supports common operations used by tests
 type Framework struct {
-	BaseName string
-	// configuration for framework's options
-	Options FrameworkOptions
-}
-
-// FrameworkOptions provide different options specfic to tests
-type FrameworkOptions struct {
-	MinNodeCount int
-	Artifacts    artifacts.ArtifactSource
+	BaseName       string
```

Comment:
Can you please update the fields with comments. What they are & how they will be useful.